Automattic/wp-calypso
# Stats: Add/day data for hourly view tabs

Related to https://github.com/Automattic/red-team/issues/268

## Proposed Changes

* Query the stats for the day to fill up the stats tabs in hourly mode

## Why are these changes being made?


* fill in empty hourly stats tabs

## Testing Instructions

* Choose 'yesterday' from date range picker and apply
* Ensure you can view Visitors, Likes and Comments data with comparison available like the screenshot below

<img width="1199" alt="image" src="https://github.com/user-attachments/assets/076a5a92-a932-418c-86a5-5c5064dff3f1">


## Pre-merge Checklist
